.github/workflows/ansible-test.yml

```diff
@@ -79,7 +79,6 @@ jobs:
           - stable-2.10
           - devel
         docker_container:
-          - ubuntu1604
```

Comment:
It's only remoed for `devel` (and upcoming `stable-2.11`), so we should remove it by an `exclude` rule (it still works for stable-2.10 and stable-2.9).